plugins/woocommerce/changelog/fix-33304-single-product-collapsed-onsale-badge-hidden

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Fix broken design of Single Product template in block themes when product had no reviews or additional info. Fix on sale badge being cut off on the Single Product template.
```

Comment:
Currently that CSS is only applied to Twenty Twenty-Two, I wonder if we should mention it instead of `block themes`. What do you think?
```suggestion
Fix broken design of Single Product template in Twenty Twenty-Two when product had no reviews or additional info. Fix on sale badge being cut off on the Single Product template.
```